app/src/components/pages/InserireVideo.jsx

```diff
@@ -0,0 +1,237 @@
+                  <Input
+                    id="videoUrl"
+                    placeholder="Inserisci il link del video"
+                    value={videoUrl}
+                    onChange={(e) => setVideoUrl(e.target.value)}
+                    className="border-[#178563] text-[#178563] placeholder-[#178563]/50"
+                  />
+                </div>
+
+                <div className="space-y-1">
+                  <Label htmlFor="video" className="text-[#178563]">File video</Label>
+                  {videoFile && (
+                    <div className="flex justify-between items-center">
+                      <span className="text-[#178563] font-semibold">{videoFile.name}</span>
+                      <button
+                        type="button"
+                        onClick={removeFile}
+                        className="text-red-600 hover:text-red-800"
+                      >
+                        X
+                      </button>
+                    </div>
+                  )}
+                  <div className="flex items-center justify-center w-full">
+                    <label
+                      htmlFor="video"
+                      className="flex flex-col items-center justify-center w-full h-40 border-2 border-dashed border-[#178563] rounded-lg cursor-pointer bg-white hover:bg-[#178563]/10"
+                    >
+                      <div className="flex flex-col items-center justify-center pt-4 pb-4">
+                        <Upload className="w-8 h-8 mb-2 text-[#178563]" />
+                        <p className="text-sm text-[#178563]">
+                          <span className="font-semibold">Clicca per caricare</span> o trascina il file qui
+                        </p>
+                        <p className="text-xs text-[#178563]">
+                          MP4, WebM o OGG (MAX. 2GB)
+                        </p>
+                      </div>
+                      <input
+                        id="video"
+                        type="file"
+                        className="hidden"
+                        accept="video/*"
+                        onChange={(e) => setVideoFile(e.target.files[0])}
+                      />
+                    </label>
+                  </div>
+                </div>
+              </div>
+
+              {videoError && <p className="text-red-600 text-lg font-semibold">{videoError}</p>}
+
+              <Button
+                type="submit"
+                className="w-full bg-[#178563] text-white rounded-full hover:bg-[#0d6145]"
+                disabled={uploading}
+              >
+                {uploading ? 'Caricamento in corso...' : 'Carica video'}
+              </Button>
+            </form>
+
+            {uploadSuccess && <p className="text-green-600 text-lg font-semibold mt-4">Il video è stato caricato con successo!</p>}
+          </div>
+        </main>
+      </div>
+    </div>
+  );
+}
```

Comment:
In questo momento non è possibile accedere a questa pagina, in quanto non è presente un comando per farlo, sarebbe opportuno inserire questo comando all'interno della pagina relativa ai contenuti video, come nella pagina di caricamento dei file, che, allo stesso modo della pagina di caricamento file, nel momento in cui si è un mentee, il bottone non verrà mostrato, altrimenti, se sono un mentore, si